pkg/nugetconfig/parse_test.go

```diff
@@ -0,0 +1,54 @@
+package nugetconfig
```

Comment:
It is better to use another package if possible.

```suggestion
package nugetconfig_test
```

Reply:
Parse method exists in nugetconfig package, hence kept it as is.

Reply:
What do you mean?

Reply:
done